cdi/plugins/org.jboss.tools.cdi.gen/META-INF/MANIFEST.MF

```diff
@@ -1,38 +0,0 @@
-Manifest-Version: 1.0

```

Comment:
This plugin was used to create big CDI projects for testing purposes. It has not been maintained for years.
We could move it to cdi/tests (it should have been placed there in the very beginning) but since we don't use/maintain it anymore we decided to move it to Slava's github workspace.